helmfile/charts/calico-felix-metrics/README.md

```diff
@@ -0,0 +1,8 @@
+# calico-felix-metrics
+
+This helm chart deploys `calico-felix-metrics`, an application that gathers metrics about the general health of Calico.
+
```

Comment:
nit: Maybe add a sentence that this chart assumes you have setup felix to expose metrics. And also add a small snippet with a command to verify that it's enabled.
Just so that people assumes that it will work as soon as they have enabled this.

Reply:
done. please check